backend/location/utility.js

```diff
@@ -0,0 +1,43 @@
+const Property = require('./property');
+
+/* Utility object stores the information about a utility location.
+ * @param name String name of the location
+ * @param boardPreset JSON with data loaded from the stored board
+ */
+class Utility extends Property {
+
+    // set up initial state
+    constructor(name, boardPreset) {
+        super(name, boardPreset);
+
+        // TODO decide if needs other features
+
+    }
+
+    /* Sees how much rent costs if this is landed on.
+     * @param numOwned int of number of other utilities owned
+     * 
+     * @return int value
+     */
+    getRent(numOwned) {
+        // let dice = amount shown on dice
```

Comment:
and even if this.rent didn't exist, you could either make a json or list to store all of the values like [10, 20, 30] or {1: 10, 2: 20, 3:30} and then just access things from the data structure instead of having all of these statements